assets/dev/js/frontend/utils/lightbox/lightbox-manager.js

```diff
@@ -87,11 +87,31 @@ export default class LightboxManager extends elementorModules.ViewModule {
 		// Detecting lightbox links on init will reduce the time of waiting to the lightbox to be display on slow connections.
 		this.elements.$links.each( ( index, element ) => {
 			if ( this.isLightboxLink( element ) ) {
+				this.maybeInsertLightboxStyle();
 				LightboxManager.getLightbox();
 
 				// Breaking the iteration when the library loading has already been triggered.
 				return false;
 			}
 		} );
 	}
+
+	maybeInsertLightboxStyle() {
```

Comment:
@Batsirai-muchareva @rami-elementor @louiswol94 

The lightbox is activated on any `a href` element on the page that links to an image, even if the link is added by code outside of Elementor.

Therefore I don't think it is good enough to handle this inside the Elementor control assets loading.

Is it safe to load the lightbox styling with JS after the DOM content was loaded?

I assume this is safe, because we don't need the lightbox to render on page load. The lightbox styling is only necessary once a link is clicked and after the Lightbox manager was activated.

Wdyt?